tests/testthat/test-replace_missing_values.R

```diff
@@ -0,0 +1,36 @@
+data <- readRDS(system.file("extdata", "test_df.RDS", package = "cleanepi"))
+test_that("replace_missing_values works", {
+  cleaned_data <- replace_missing_values(
+    data           = data,
+    target_columns = "sex",
+    na_strings     = "-99"
+  )
+  expect_s3_class(cleaned_data, "data.frame")
+  expect_false("-99" %in% cleaned_data[["sex"]])
+  expect_true(anyNA(cleaned_data[["sex"]]))
```

Comment:
Could you add a test that shows that where a previous cell had `-99`, it is now `NA`?

It'll be something like `expect_true(all(is.na(cleaned_data$sex[data$sex == "-99"]))`

Reply:
nice one - will account for this